### PR TITLE
Support for lambda-proxy integration.

### PR DESCRIPTION
**Pull request by gertjvr**
Taking inspiration from #117, only thing missing was the requestContext and some others 
-  not setting up a request / response template unless lambda integration was specify.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -367,7 +369,7 @@ class Offline {
             const contentType = request.mime || defaultContentType;
             // default request template to '' if we don't have a definition pushed in from serverless or endpoint
             let requestTemplate = '';
-            if (typeof requestTemplates !== 'undefined') {
+            if (typeof requestTemplates !== 'undefined' && endpoint.integration === 'lambda') {
```
Comment (lalifraco-devspark) [inline src/index.js]: We should use the integration variable defined in line 277 here.

**Comment by leonardoalifraco** [comment]
This is great and helpful, thanks!
Only that little change to use the integration variable (which has a default value for lambda-proxy) and I'll merge it.

**Comment by gertjvr** [comment]
Thanks for reviewing it

**Comment by leonardoalifraco** [comment]
👍

**Comment by gertjvr** [comment]
now to get your pr merged

**Comment by gertjvr** [comment]
did I base my pr on the right branch? can't see my changes in your pr

**Comment by leonardoalifraco** [comment]
Not actually. It should have been to the leonardoalifraco:lambda-proxy-event branch. But I've just merged it and your changes are now in the PR.

**Pull request merged**